test/test_001.py

```diff
@@ -148,18 +146,77 @@
             '"Test Project 2-ff0e78e", "Test Project 2-ff0e78e", "nerc_openstack"\n',
         ],
     },
+    "2": {
+        "coldfront_data": [
+            {
+                "id": 1,
+                "project": {
+                    "id": 11,
+                    "title": "Test-Project-1",
+                    "pi": "rbb@some.edu",
+                    "description": "Experiment to run Autodesk Maya on GPU node",
+                    "field_of_science": "Visualization, Graphics, and Image Processing",
+                    "status": "New",
+                },
+                "description": None,
+                "resource": {"name": "NERC", "resource_type": "OpenStack"},
+                "status": "Active",
+                "attributes": {
+                    "Allocated Project ID": "2e68e6bf3ba84d96b77ff9cf8f1fffff",
+                    "Allocated Project Name": "Test Project 1-ff0e78e",
+                },
+            },
+        ],
+        "keycloak_data": [
+            {
+                "id": "7c6a4bea-37f4-46a3-8c52-4a93731fffff",
+                "createdTimestamp": 1672154253164,
+                "username": "rbb@some.edu",
+                "enabled": True,
+                "totp": False,
+                "emailVerified": True,
+                "firstName": "R",
+                "lastName": "B",
+                "email": "rbb@some.edu",
+                "attributes": {
+                    "cilogon_idp_name": ["Some University"],
+                    "mss_research_domain": ["engineering"],
+                },
+                "disableableCredentialTypes": [],
+                "requiredActions": [],
+                "notBefore": 0,
+                "access": {
+                    "manageGroupMembership": True,
+                    "view": True,
+                    "mapRoles": True,
+                    "impersonate": True,
+                    "manage": True,
+                },
+            },
+        ],
+        "hierarchy.csv": [
+            '"3","Some University",\n',
+            '"4","Some University - engineering","3"\n',
+            '"5","R B <rbb@some.edu>","4"\n',
+        ],
+        "group.csv": [
+            '"Test Project 1-ff0e78e", "5"\n',
+        ],
+        "names.csv": [
+            '"Test Project 1-ff0e78e", , "Test-Project-1 - Test Project 1-ff0e78e"\n',
+        ],
+        "pi2project.csv": [
+            '"Test Project 1-ff0e78e", "Test Project 1-ff0e78e", "nerc_openstack"\n',
+        ],
+    },
 }
 
 
 def test_000(mocker):
```

Comment:
The test name should describe the purpose of the test (so that the output of a test run is informative). Names like `test_widget_created_successfully` or `test_cannot_create_widget_when_quota_succeeded` are examples of more informative names.